nemo/utils/config_utils.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+from dataclasses import is_dataclass
+
+from omegaconf import DictConfig, OmegaConf, open_dict
+
+from nemo.core.config.modelPT import ModelPTConfig
+
+
+def update_model_config(model_cls: ModelPTConfig, update_cfg: DictConfig, drop_missing_subconfigs: bool = True):
```

Comment:
Can we merge these functions into hydra_runner()?

Reply:
I would keep this function and suggest using it inside hydra_runner. This will make this functionality available for those not wishing to use hydra_runner decorator.

Reply:
Agreed, but won't this affect TLT scripts? I would urge @tkornuta-nvidia to comment on this - can you pull this branch and run TLT tests once after updating hydra runner in order to ensure it has no effect on TLT ?